client/src/components/Settings/SettingsView.res

```diff
@@ -84,7 +100,11 @@ let html = (m: AppTypes.model): Html.html<msg> => {
 
   Html.div(
     list{
-      Attrs.class("settings modal-overlay"),
+      Attrs.class(
+        %twc(
+          "z-[100] fixed top-0 left-0 w-full h-full bg-[#484848cc] flex items-center justify-center"
```

Comment:
I think there's a feature to allow us do opacity directly: https://tailwindcss.com/docs/background-color#changing-the-opacity

So we could use `bg-grey1/80` I think?